packages/ubus-lime-utils/files/usr/libexec/rpcd/lime-utils-admin

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env lua
+--[[
+  Copyright (C) 2020 LibreMesh.org
+  This is free software, licensed under the GNU AFFERO GENERAL PUBLIC LICENSE Version 3
+
+  Copyright 2020 Santiago Piccinini <spiccinini@altermindi.net>
+]]--
+
+local ubus = require "ubus"
+local json = require 'luci.jsonc'
+local utils = require 'lime.utils'
+
+local conn = ubus.connect()
+if not conn then
+    error("Failed to connect to ubus")
+end
+
+local function set_root_password(msg)
```

Comment:
@spiccinini this is the ubus method we want to expose in order to let a limeapp user to change the shared-password?
In that case we should be writting the root_secret option at lime-community right?

Reply:
Great catch :)

Reply:
I've addressed this in #701